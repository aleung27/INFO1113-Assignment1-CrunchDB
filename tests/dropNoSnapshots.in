DROP 1
SET a 2
SNAPSHOT
DROP 0
LIST SNAPSHOTS
