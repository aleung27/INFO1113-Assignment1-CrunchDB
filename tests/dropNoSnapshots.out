> no such snapshot

> ok

> saved as snapshot 1

> no such snapshot

> 1

